int64 valor1
int64 valor2
---
int64 resultado